domains/transfer-agreements/TransferAgreements.API/API/Migrations/ApplicationDbContextModelSnapshot.cs

```diff
@@ -1,148 +1,177 @@
-﻿// <auto-generated />

```

Comment:
Since all lines are changed here, I think that there is a change of line-endings or encoding. Could you check and potentially fix line-endings and encodings in this file?

Reply:
it is lf and utf-8 now, so that is good.